### ISSUE TITLE
xyplot save individual image with name of the x and y?

### ISSUE THREAD

**Issue by thexqin** [yolain/ComfyUI-Easy-Use#992]
hi yolain,

thank you for such great tool. i am wondering is there a way to output the invidual images but rename them so that each file has the correct name of the x and y? for instance i am testing 9 schedulers + 9 samplers, resulting in 81 images. but for each image output, i have no idea what scheduler + sampler it has used.

thank you.

<img width="1111" height="671" alt="Image" src="https://github.com/user-attachments/assets/f623f31d-a0ff-4439-9dd7-31ab3b2499fb" />
<img width="376" height="493" alt="Image" src="https://github.com/user-attachments/assets/c84ca476-e1aa-4e97-82d1-21d98ad41c57" />

**Comment by thexqin** [comment]
is there something like this already implented in the repo?

```
import torch

class XYNameExtractor:
    @classmethod
    def INPUT_TYPES(s):
        return {
            "required": {
                "x_metadata": ("X_Y",),
                "y_metadata": ("X_Y",),
                "images": ("IMAGE",),
            },
        }

    RETURN_TYPES = ("STRING",)
    OUTPUT_IS_LIST = (True,)
    FUNCTION = "get_names"
    CATEGORY = "Custom/XYPlot"

    def get_names(self, x_metadata, y_metadata, images):
        # Extract labels from the Easy-Use X_Y dictionary
        x_values = x_metadata.get("values", "").split("; ")
        y_values = y_metadata.get("values", "").split("; ")
        
        num_x = len(x_values)
        num_images = images.shape[0]  # Get batch size from image tensor
        
        results = []
        
        for i in range(num_images):
            # The logic: Y is outer loop, X is inner loop
            # Image 0, 1, 2... are X1Y1, X2Y1, X3Y1...
            x_idx = i % num_x
            y_idx = i // num_x
            
            # Boundary check
            x_label = x_values[x_idx] if x_idx < len(x_values) else "unknown_x"
            y_label = y_values[y_idx] if y_idx < len(y_values) else "unknown_y"
            
            # Create a filename-friendly string
            clean_name = f"{x_label}_{y_label}_{i:03d}"
            results.append(clean_name)
            
        return (results,)

NODE_CLASS_MAPPINGS = {"XYNameExtractor": XYNameExtractor}
NODE_DISPLAY_NAME_MAPPINGS = {"XYNameExtractor": "XY Filename Generator"}
```
